cluster_cleanup.sh

```diff
@@ -7,6 +7,8 @@ source lib/common.sh
 # Delete cluster
 if [ "${EPHEMERAL_CLUSTER}" == "kind" ]; then
   sudo su -l -c "kind delete cluster  || true" "${USER}"
+  # Kill and remove the running ironic containers
+  "$BMOPATH"/tools/remove_local_ironic.sh
```

Comment:
That can be problematic, in case you did not run the 03* before calling make clean. Either you should ensure that this will not fail, or you should not call it if the executable is not there

Reply:
Humm... this is getting complicated!!

Reply:
How do you think about the current change?